test/functional/unattended_controller_test.rb

```diff
@@ -243,7 +243,7 @@ class UnattendedControllerTest < ActionController::TestCase
     @request.env["REMOTE_ADDR"] = @ub_host.ip
     @ub_host.create_token(:value => token, :expires => Time.now + 5.minutes)
     get :provision, {'token' => @ub_host.token.value }
-    assert @response.body.include?("#{Setting[:unattended_url]}:443/unattended/finish?token=#{token}")
+    assert @response.body.include?("/unattended/finish?token=#{token}")
```

Comment:
Oh, what's this?  Is the full URL not getting included or is there an issue with the value of the unattended URL or something?